### DISCUSSION TITLE
the button save or restart in Admin/Settings without function

### DISCUSSION THREAD

**Discussion by Erika-epr** [ether/etherpad-lite#5243] [Q&A]
I make changes to the settings.json (admin/settings) via the web interface and press the save button,
the button does not react. It is the same with the restart etherpad button.
Does anyone have any idea what I am doing wrong?

Node-Version: v12.18.2
Etherpad_Lite: Versionsnummer: 1.8.14
Git sha: 571b37b

Etherpad runs as a service:
Environment=NODE_ENV=production
ExecStart=/usr/bin/node /xxx/xxx/etherpad-lite/node_modules/ep_etherpad-lite/node/server.js

**Comment by JohnMcLear** [comment]
Just FYI we have automated test coverage in for this so it should be working.

Copy of latest test: https://github.com/ether/etherpad-lite/runs/3961728692?check_suite_focus=true
Video of test runner: https://app.saucelabs.com/tests/72c0aff9c54a4fb5ba50fb3f3bea14c0

**Comment by Erika-epr** [comment]
What else could it be?

I used the latest version of chrome, but the buttons don't react with firefox either.

**Comment by JohnMcLear** [comment]
If you hit f12 do you see any errors in developer console?

**Comment by Erika-epr** [comment]
Hello,
yes, I see the following errors:

![Bildschirmfoto 2021-10-22 um 16 01 29](https://user-images.githubusercontent.com/68369740/138467405-b37f3a66-4978-45d9-acf8-22126d1cda55.png)
![Bildschirmfoto 2021-10-22 um 16 01 17](https://user-images.githubusercontent.com/68369740/138467409-7041142d-6a43-47a7-b46b-6ca67399b7bb.png)

**Reply by Erika-epr** [reply]
now I have added the following to the Apache config and get this error every second:
    SSLProxyEngine On
      ProxyVia On
      ProxyRequests Off
      ProxyPreserveHost on

      <Location /socket.io>
        RewriteEngine On
        RewriteCond %{QUERY_STRING} transport=websocket    [NC]
        RewriteRule /(.*) ws://localhost:9001/socket.io/$1 [P,L]
        ProxyPass http://localhost:9001/socket.io retry=0 timeout=30
        ProxyPassReverse http://localhost:9001/socket.io
      </Location>
![Bildschirmfoto 2021-10-22 um 16 27 24](https://user-images.githubusercontent.com/68369740/138471762-9558f3f0-a5d8-4b1f-b6c4-d779c1673b12.png)

**Reply by rhansen** [reply]
Your reverse proxy is misconfigured if you are getting 404 errors for `/socket.io`.

**Comment by JohnMcLear** [comment]
Can you replicate this bug if you interact with Etherpad directly?

**Comment by Erika-epr** [comment]
excuse me, i don't understand what i'm supposed to do?

**Comment by JohnMcLear** [comment]
Assuming you are using Apache to access Etherpad?  If so, access directly ie use port 9001

**Comment by Erika-epr** [comment]
Yes, we go immediately to Port 9001
